notifications.bs

```diff
@@ -343,20 +394,26 @@ language tag. Validity or well-formedness are not enforced. [[!LANG]]
 
 <h3 id=activating-a-notification>Activating a notification</h3>
 
-<p>When a <a lt="concept notification">notification</a> <var>notification</var>
-is activated by the user, assuming the underlying notification platform supports
-activation, the user agent must (unless otherwise specified) run these steps:
+<p>When a <a lt="concept notification">notification</a> <var>notification</var>,
+or one of its <a>actions</a>, is activated by the user, assuming the underlying
+notification platform supports activation, the user agent must (unless otherwise
+specified) run these steps:
 
 <ol>
   <li><p>If <var>notification</var> is a <a>persistent notification</a>, run
   these substeps:
 
   <ol>
+    <li><p>If one of <var>notification</var>'s <a>actions</a> was activated by
+    the user, let <var>action</var> be that action's <a>action name</a> string.
+
+    <li><p>Otherwise, let <var>action</var> be the empty string.
```

Comment:
I think I would prefer reversing these two steps and explicitly setting setting `<var>action</var>` if an action happened to be activated. Using "let" twice feels somewhat unclean (although I realize it's currently common in the HTML standard).